Makefile

```diff
@@ -297,7 +297,7 @@ ifneq ($(BUILD),local)
 	@docker build --build-arg GOVER=$(GOVER) --build-arg USER=$(USER) --build-arg GROUP=$(GROUP) \
                       --build-arg UID=$(UID) --build-arg GID=$(GID) \
                       $(DOCKER_HTTP_PROXY) $(DOCKER_HTTPS_PROXY) $(DOCKER_NO_PROXY) $(DOCKER_ALL_PROXY) \
-                      -t $@ build-tools/src/scripts > /dev/null
+                      -t $@ build-tools/src/scripts
```

Comment:
Any particular reason you're making it more verbose by removing >/dev/null ?

I mean -- I suppose we need to introduce a full verbose mode with something like make V=1 ... but that would be a separate PR.

Reply:
sorry, That is for debugging purpose only. I will change back

Reply:
Solved, and re-pushed the patches in the same PR 354. Please review again